lib/i18n/traditional-chinese.js

```diff
@@ -0,0 +1,107 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Genie
+//
+// Copyright 2018-2019 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Giovanni Campagna <gcampagn@cs.stanford.edu>, Johnny Hsu <johnny.chhsu01@gmail.com>
+//
+// See COPYING for details
+"use strict";
+
+function replaceMeMy(sentence) {
+    sentence = sentence.replace(/\b((?!(?:通知|告知)\b)[a-zA-Z0-9]+) me\b/g, '$1 他們');
+
+    return sentence.replace(/\b(我|我的)\b/g, (what) => {
+        switch(what) {
+        case '我':
+            return '他';
+        case '我的':
+            return '他們的';
+        default:
+            return what;
+        }
+    });
+}
+
+function postprocessSynthetic(sentence, program) {
+    if (program.isProgram && program.principal !== null)
+        sentence = replaceMeMy(sentence);
+
+    return sentence.replace(/ (新的|新) (他們的|我的|一個) /, (_new, what) => ` ${what} ${_new} `)
+        .trim();
```

Comment:
This postprocessing step is for artifacts in the English templates, I don't know if you'll need it for Chinese - you won't know until you write the templates, really.

Reply:
We plan to use a blood pressure measuring machine as our demo device for Chinese, I think we can come up with some template with it first then see what will we need.